xml/pop_banners.xml

```diff
@@ -216,8 +216,56 @@ matched against these patterns to fingerprint POP3 servers.
     <param pos="0" name="service.family" value="Courier MTA"/>
     <param pos="0" name="service.product" value="Courier POP"/>
   </fingerprint>
-  <!--
-
+  <fingerprint pattern= "^CMailServer ([\d\.]+) POP3 Service Ready$">
+    <description>CMailServer</description>
+    <example service.version="5.0.0">CMailServer 5.0.0 POP3 Service Ready</example>
+    <param pos="0" name="service.product" value="CMailServer"/>
+    <param pos="0" name="service.vendor" value="Youngzsoft"/>
+    <param pos="0" name="os.family" value="Windows" />
+    <param pos="0" name="os.vendor" value="Microsoft"/>
+    <param pos= "1" name="service.version"/>
+  </fingerprint>
+  <fingerprint pattern= "^POP3 Bigfoot v(\d\.\d) server ready$">
+    <description>POP3 Bigfoot server</description>
+    <example service.version="1.0">POP3 Bigfoot v1.0 server ready</example>
+    <param pos="0" name="service.vendor" value="Bigfoot"/>
+    <param pos="0" name="service.family" value="Bigfoot Email Tools"/>
+    <param pos="0" name="service.product" value="Bigfoot Email Tools"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+  <fingerprint pattern="^CCProxy ([\d.]+) POP3 Service Ready$">
+    <description>CCProxy POP3 server</description>
+    <example service.version="8.0" >CCProxy 8.0 POP3 Service Ready</example>
```

Comment:
really minor, but here and and in a few other places you've got trailing space towards the end of your opening elements (in this case, in `example`.